tests/integration-tests/src/twoliter_update.rs

```diff
@@ -1,13 +1,15 @@
+use crate::twoliter_build::copy_project_to_temp_dir;
```

Comment:
Thanks for these improvements to the integration tests! Do you mind putting them into a separate commit?